src/platform_impl/windows/window.rs

```diff
@@ -631,7 +629,10 @@ impl CoreWindow for Window {
                 .lock()
                 .unwrap()
                 .mouse
-                .set_cursor_flags(window.hwnd(), |f| f.set(CursorFlags::GRABBED, confine))
+                .set_cursor_flags(window.hwnd(), |f| {
+                    f.set(CursorFlags::GRABBED, grabbed);
```

Comment:
```suggestion
                    f.set(CursorFlags::GRABBED, mode == CursorGrabMode::Confined);
```

Reply:
i'll delete the `let (grabbed, locked) =` block above and use

```suggestion
                    f.set(
                        CursorFlags::GRABBED,
                        mode == CursorGrabMode::Confined || mode == CursorGrabMode::Locked,
                    );
```

in `refresh_os_cursor` all the clipping is contingent on `GRABBED` so it needs to be set for both modes.